numbits.go

```diff
@@ -0,0 +1,41 @@
+package quamina
+
+import (
+	"math"
+)
+
+// float64 are stored as (sign | exponent | mantissa)
+// with 1 bit sign, 11 bits exponent, 52 bits mantissa
+const ()
```

Comment:
Oops, editing error here didn't notice the empty const() and my introduction needs to move up to the top.